### ISSUE TITLE
Allow to define $prefix for used CSS variables for Bootstrap5 theme

### ISSUE THREAD

**Issue by kevinpapst** [fullcalendar/fullcalendar#6897]
### Checklist

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

A couple of us are reporting missing borders in the Bootstrap5 theme.
That is likely related to the use of the Bootstrap variable `--bs-gray-400` which Bootstrap themes can but do not need to export.

https://github.com/fullcalendar/fullcalendar/blob/bb20a0b080bfac9341a5a6ba903afc596a91b0d2/packages/bootstrap5/src/main.css#L13-L13

Bootstrap added the [new $prefix build variable](https://getbootstrap.com/docs/5.2/customize/css-variables/#prefix) and that could be nice to support, so we can define e.g. in ES6 builds which prefix should be used.

The alternative solution is that we have to redefine the variables in our own SCSS:
```
    --bs-gray-200: #e2e8f0;
    --bs-gray-400: #94a3b8;
```
but I'd like to re-use the existing `--mythmee-gray-400` variable.

**Comment by arshaw** [comment]
I thinking a simpler solution might be to simply hardcode the gray hex values. They can't be all that different from theme-to-theme.

**Comment by kevinpapst** [comment]
Agree. Re-using this variable for purple would be weird, though I have not much experience with accessibility and if there might be themes that emphasize the color. 

Anyway, hardcoding ist still better than not having any border 😁
